DFC.ServiceTaxonomy.JobProfiles.Module/ServiceBusHandling/Converters/Helper.cs

```diff
@@ -65,7 +65,7 @@ public static async Task<IEnumerable<string>> GetRelatedSkillsAsync(dynamic cont
                     var contentItems = await contentManager.GetAsync(idList);
                     foreach (var item in contentItems)
                     {
-                        contentItemNames.Add(item.ContentItem.Content.SOCSkillsMatrix.RelatedSkill.Text.ToString().GetSlugValue());
+                        contentItemNames.Add(((string)item.ContentItem.Content.SOCSkillsMatrix.RelatedSkill.Text).GetSlugValue());
```

Comment:
Are there any other places which use .ToString instead of (string) cast? could you please find them all and change them at the same time (if there are any)?

Reply:
no there are none